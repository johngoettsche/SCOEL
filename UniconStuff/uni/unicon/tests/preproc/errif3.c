/* error message test: hex escape too large in a condition */

#if '\x1ff'
#endif
