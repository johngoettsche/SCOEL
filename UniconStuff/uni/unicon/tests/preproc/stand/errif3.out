pp: File errif3.c; Line 3: hex escape sequece larger than a character
